flink-yarn/src/test/java/org/apache/flink/yarn/YarnFileStageTest.java

```diff
@@ -200,13 +201,23 @@ static void testCopyFromLocalRecursive(
 			while (targetFilesIterator.hasNext()) {
 				LocatedFileStatus targetFile = targetFilesIterator.next();
 
-				try (FSDataInputStream in = targetFileSystem.open(targetFile.getPath())) {
-					String absolutePathString = targetFile.getPath().toString();
-					String relativePath = absolutePathString.substring(workDirPrefixLength);
-					targetFiles.put(relativePath, in.readUTF());
-
-					assertEquals("extraneous data in file " + relativePath, -1, in.read());
-				}
+				int retries = 5;
```

Comment:
I wonder if this should be a magic number or better something that can be configured?